<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>879</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>879</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: url(data/herbe.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="newWave">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>NOUVELLE VAGUE</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>-10</y>
      <width>41</width>
      <height>591</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxCaracteristics">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>410</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Caractéristiques</string>
    </property>
    <widget class="QPushButton" name="up_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Améliorer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sell_button">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Vendre</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCaracteristics">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Type: X, Lvl : Y, Range: Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxTowers">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Créer une tour</string>
    </property>
    <widget class="QPushButton" name="waterTowers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>EAU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stoneTowers">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PIERRE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paintTowers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PEINTURE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="petanqueTowers">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PETANQUE</string>
     </property>
    </widget>
    <widget class="QLabel" name="choice">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>120</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vous avez choisi: </string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelMoney">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Crédits :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLives">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>210</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vies :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadMap">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CHARGER MAP</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="terrain">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>0.000000000000000</width>
      <height>0.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QLCDNumber" name="textMoney">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="textLives">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
